src/utils/daemon.js

```diff
@@ -1,11 +1,19 @@
 import IPFSFactory from 'ipfsd-ctl'
-import { showConnFailureErrorMessage } from './errors'
 import logger from './logger'
+import fs from 'fs-extra'
+import { join } from 'path'
 import { app } from 'electron'
 import { execFileSync } from 'child_process'
 import findExecutable from 'ipfsd-ctl/src/utils/find-ipfs-executable'
 
-function repoFsck (path) {
+async function cleanup (path) {
+  logger.info(`Entering cleanup stage`)
+
+  if (!await fs.pathExists(join(path, 'config'))) {
+    logger.info(`'config' file does not exist. Aborting cleanup`)
```

Comment:
We should show a custom error message here, as the default one asks the user if they want to report the error, and we don't want that. We want to explain that the api file and lack of config file mean we can't start the app.

Reply:
Why? In this case there is no `config` file, so ipfsd-ctl will connect to a remote node. Unless we want to prevent Desktop from connecting to a remote node. Is that so?